cpp/Platform.Collections/Platform.Collections.h

```diff
@@ -11,6 +11,8 @@
 #include <span>
 #include <iostream>
 #include <numeric>
+#include <stack>./cpp/build_and_test_gitpod.sh
```

Comment:
```suggestion
#include <stack>
```

Reply:
But I’m not sure we need to include stack header into our library by default. If it is needed in test it should be included there.